geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -1529,6 +1535,12 @@
   public static final String IMPORT_DATA__FILE = "file";
   public static final String IMPORT_DATA__FILE__HELP =
       "File from which the imported data will be read. The file must have an extension of \".gfd\".";
+  public static final String IMPORT_DATA__DIR = "dir";
+  public static final String IMPORT_DATA__DIR__HELP =
+      "Directory from which all data files (\".gfd\") will be read. Required if parallel set to true. Overrides value of \"--file\" if both specified";
```

Comment:
What is the value to the user of having --dir override --file? Would it be better if both of these are non-null to throw an error?

Reply:
Having --dir override --file was to ensure that the command would function if both were added and to give that a defined outcome. Having that fail with an exception is also fully valid: we just need to determine which experience we think is better / least surprising to a user.